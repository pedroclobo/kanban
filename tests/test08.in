t 5 desenho
t 12 desenvolvimento
t 8 testes
l
q
