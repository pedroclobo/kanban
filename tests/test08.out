task 1
task 2
task 3
1 TO DO #5 desenho
2 TO DO #12 desenvolvimento
3 TO DO #8 testes
